logging/eventrouter/eventrouter.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: v4m-eventrouter
+  namespace: "logging"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: v4m-eventrouter
+rules:
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: v4m-eventrouter
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: eventrouter
+subjects:
+- kind: ServiceAccount
+  name: eventrouter
+  namespace: "logging"
```

Comment:
Shouldn't "logging" be "__LOG_NS__" here?

Reply:
I originally hard-coded logging and was going to string replacement using **sed**.  The next pull request will address this.